src/main/java/uk/ac/ebi/eva/evaseqcol/entities/SeqColComparisonResultEntity.java

```diff
@@ -52,4 +52,10 @@ public void putIntoElements(String elementName,String key, Object value) {
         SortedMap<String, Object> elementsMap = elements.get(elementName);
         elementsMap.put(key, value);
     }
+
+    /**
+     * Return the value of the given attributeName under the given elementName*/
+    public Boolean getElement(String elementName, String attributeName) {
+        return (Boolean) elements.get(elementName).get(attributeName);
```

Comment:
The values can be Integer or Boolean, so should probably check before casting... Also, I don't see where we actually use this method.